apps/condo/domains/user/schema/SupportMessagesForwardingService.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Generated by `createservice user.SupportMessagesForwardingService`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/user/access/SupportMessagesForwardingService')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const { MESSAGE_FORWARDED_TO_SUPPORT } = require('@condo/domains/notification/constants/constants')
+const { SUPPORT_EMAIL } = require('@condo/domains/common/constants/requisites')
+const { get } = require('lodash')
+const { LOCALES } = require('@condo/domains/common/constants/locale')
+const { Resident } = require('@condo/domains/resident/utils/serverSchema')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const { v4: uuid } = require('uuid')
+const dayjs = require('dayjs')
+
+const FORWARDED_EMAILS_ATTACHMENTS_FILE_FOLDER_NAME = 'forwarded-emails-attachments'
+const adapter = new FileAdapter(FORWARDED_EMAILS_ATTACHMENTS_FILE_FOLDER_NAME)
+
+const SupportMessagesForwardingService = new GQLCustomSchema('SupportMessagesForwardingService', {
+    types: [
+        {
+            access: true,
+            type: `enum SupportMessagesForwardingLang { ${Object.keys(LOCALES).join(' ')} }`,
+        },
+        {
+            access: true,
+            type: 'input SupportMessagesForwardingInput { dv: Int!, sender: JSON!, text: String!, email: String, attachments: [Upload], os: String!, appVersion: String!, lang: SupportMessagesForwardingLang!, meta: JSON! }',
+        },
+        {
+            access: true,
+            type: 'type SupportMessagesForwardingOutput { id: String!, status: String! }',
+        },
+    ],
+
+    mutations: [
+        {
+            access: access.canSupportMessagesForwarding,
+            schema: 'supportMessagesForwarding(data: SupportMessagesForwardingInput!): SupportMessagesForwardingOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const { data } = args
+                const { dv, sender, text, email: emailFrom, attachments = [], os, appVersion, lang, meta } = data
+
+                const user = get(context, ['req', 'user'])
+
+                const attachmentsData = await Promise.all(attachments)
```

Comment:
It looks like a high memory usage

Reply:
Field `attachments` contains only streams of attached files. I suppose we're not putting the whole file into memory.